core/thegraph/state.go

```diff
@@ -131,6 +139,25 @@ func (ics *indexedChainState) GetIndexedOperatorState(ctx context.Context, block
 		return nil, err
 	}
 
+	// Detect missing operators
+	operatorSeen := make(map[core.OperatorID]struct{})
+	for _, quorumOperators := range operatorState.Operators {
+		for operatorID := range quorumOperators {
+			if indexedOperators[operatorID] == nil {
+				return nil, fmt.Errorf("operator %s not found in indexed state", operatorID.Hex())
+			}
+			operatorSeen[operatorID] = struct{}{}
+		}
+	}
+
+	// Filter out the operators who are not part of any quorum. This can happen if the operator re-registered
+	// after the reference block number.
+	for operatorID := range indexedOperators {
```

Comment:
wouldn't it apply to the new operators joining after the referenceBlockNumber?

Reply:
Yeah, this should say registered or re-registered.